scripts/loadDbSettings.ts

```diff
@@ -77,9 +87,18 @@ const deleteByName = (db: Database, name: string) =>
     max: 3,
   });
 
+  let databaseIdPromise: Promise<void>;
+  if (mode === "prod") {
+    databaseIdPromise = Promise.resolve();
+  } else if (mode === "staging") {
+    databaseIdPromise = setDatabaseId(db, randomId(), "ea-staging");
```

Comment:
I may be being still-sleepy and confused here, but why are we setting the staging db / why are we setting it to something random? That way it seems to me when you try to connect to staging locally, you're going to get an error?

Reply:
Oh whoops, you already merged the relevant PR.

Reply:
Yeah - I accidentally merged it here instead of into `master` 🙃. It's fine though, because this is basically read to merge too anyway.

This is just matching the behaiour that currently exists in ForumCredentials. We're setting the value to `ea-staging` - the `randomId()` is for the `_id` field.

Reply:
oh, jk